client/my-sites/stats/stats-post-detail/index.jsx

```diff
@@ -37,16 +38,14 @@ export default React.createClass( {
 	render() {
 		let title;
 
-		if ( this.props.postViewsList.response.post && this.props.postViewsList.response.post.post_title ) {
-			title = this.translate( 'Stats for %(posttitle)s', {
-				comment: 'Title of the individual post stats page.',
-				args: {
-					posttitle: this.props.postViewsList.response.post.post_title
-				}
-			} );
-		}
+		const post = this.props.postViewsList.response.post;
+		const postOnRecord = post && post.post_title !== null;
 
-		if ( this.props.postViewsList.isError() ) {
+		if ( postOnRecord && ! this.props.postViewsList.isError() ) {
```

Comment:
I took out the `this.props.postViewsList.isError()` check and could not trigger any odd behavior.

@jonathansadowski: Looks like you included this originally -- do you know if it's still needed?